README.md

```diff
@@ -34,21 +34,18 @@ Flutter öğrenmeye başlayan pek çok insan nereden başlayacağını veya nas
 Flutter ile ilgili Türkçe makaleler kategorilerine göre ayrılmış halde burada bulunabilir.
 
 #### Dart
-- [Dart Const Eğitici İçerik— Bilmeniz Gereken Her şey(Const İfadeler, Standart Örnekler ve Daha Fazlası)](https://medium.com/flutter-t%C3%BCrkiye/dart-const-e%C4%9Fitici-i%CC%87%C3%A7erik-bilmeniz-gereken-her-%C5%9Fey-const-i%CC%87fadeler-standart-%C3%B6rnekler-ve-daha-fe243f590059) - Beyza Sunay Güler'in Matt Rešetár (Reso Coder)'ın makalesinden çevirisi
 - [Dart dilinde Uzantı Metotları(Extension Methods)](https://medium.com/flutter-i%CC%87zmir/dart-dilinde-uzant%C4%B1-metotlar%C4%B1-extension-methods-912158f92fa1) - Mirkan Çalışkan
 - [Dart Programlama Dili](https://medium.com/@halilozel1903/dart-programlama-dili-eeafb64ad300) - Halil Özel
 
 #### Flutter'a Giriş
 - [Flutter ile Güzel Kullanıcı Arayüzü Oluşturmak](https://medium.com/flutter-t%C3%BCrkiye/flutter-ile-g%C3%BCzel-kullan%C4%B1c%C4%B1-aray%C3%BCz%C3%BC-olu%C5%9Fturmak-2b8a3c4c7b17) - Mustafa Türkmen'in Flutter codelab çevirisi
 - [Flutter Nedir ? Devrim Etkisi Oluşturur Mu?](https://medium.com/flutter-t%C3%BCrkiye/flutter-nedir-devrim-etkisi-olu%C5%9Fturur-mu-3f9c0b8bd251?source=collection_detail----f3fe55e37b59-----26-----------------------) - Yasin İlhan
 - [Flutter ile ilk uygulamanı yazmaya hazır mısın? (Part 2)](https://medium.com/flutter-t%C3%BCrkiye/flutter-ile-i%CC%87lk-uygulaman%C4%B1-yazmaya-haz%C4%B1r-m%C4%B1s%C4%B1n-part-2-ac97b0ee4baf) - Ajda Akter'in Flutter codelab çevirisi
-- [Flutter öğrenmek istiyorum, nereden başlamalıyım](https://medium.com/flutter-t%C3%BCrkiye/flutter-%C3%B6%C4%9Frenmek-i%CC%87stiyorum-nas%C4%B1l-ba%C5%9Flamal%C4%B1y%C4%B1m-650427b18cfe) - Beyza Sunay Güler'in Pooja Baumik'in makalesinden çevirisi
 - [Flutter ile ilk uygulamanı yazmaya hazır mısın? (Part 1)](https://medium.com/flutter-t%C3%BCrkiye/flutter-ile-i%CC%87lk-uygulaman%C4%B1-yazmaya-haz%C4%B1r-m%C4%B1s%C4%B1n-part-1-f2833af78685) - Ajda Akter'in Flutter codelab çevirisi
 - [Flutter Nedir ?](https://medium.com/@halilozel1903/flutter-nedir-8a3cb33fdb9d) - Halil Özel
 - [Android Studio’ya Flutter Kurulumu](https://medium.com/@halilozel1903/android-studioya-flutter-kurulumu-73f7d4cefde) - Halil Özel
 - [Flutter’ın Mimarisi ve Dart’ın Flutter üzerindeki etkisi](https://medium.com/flutter-t%C3%BCrkiye/flutter%C4%B1n-mimarisi-ve-dart-%C4%B1n-flutter-%C3%BCzerindeki-etkisi-b9b652d0525) - Muhammed Salih Güler
 - [Flutter’ da pro düzeye ulaşmak için gerekli adımlar](https://medium.com/@hubders/flutter-da-pro-d%C3%BCzeye-ula%C5%9Fmak-i%C3%A7in-gerekli-ad%C4%B1mlar-faa6837a03de) - DersHub
-- [Flutter için kod örnekleri.] (onursahin.net/flutter) - Onur Şahin @imonursahin
```

Comment:
Why you deleted this?

Reply:
You can change to https://onursahin.net/flutter

Reply:
I searched but couldn't find it. I will add this.